azure-iot-device/samples/advanced-hub-scenarios/send_to_output_x509.py

```diff
@@ -0,0 +1,64 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import os
+import uuid
+from azure.iot.device.aio import IoTHubModuleClient
+from azure.iot.device.iothub import Message
+from azure.iot.device.common import X509
+import logging
+import asyncio
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+messages_to_send = 10
+
+
+async def main():
+    hostname = os.getenv("HOSTNAME")
+
+    # The device having a certain module that has been created on the portal
```

Comment:
There's a conversation to be had about what samples we want to actually have, and how we want to structure them - we don't want to drown customers in tons of redundant samples that are subtly different. But for now this is fine.